compiler/src/iree/compiler/Codegen/LLVMGPU/Passes.cpp

```diff
@@ -77,7 +77,8 @@ static llvm::cl::opt<bool> clLLVMGPUEnablePrefetch(
 
 llvm::raw_ostream &operator<<(llvm::raw_ostream &os,
                               const LLVMGPUPipelineOptions &options) {
-  return os << "{" << "enableReduceSharedMemoryBankConflicts = "
+  return os << "{"
+            << "enableReduceSharedMemoryBankConflicts = "
```

Comment:
Why is this changed?

Reply:
adapting to changes in the data structure (LLVMGPUPipelineOptions).

Reply:
I think this just came from a different clang-format version. Same thing happened to me.

Reply:
Yep, clang-format wants this line unchanged: https://github.com/iree-org/iree/actions/runs/9472784149/job/26099808430?pr=17645#step:4:120

You can set up pre-commit to run formatting automatically or install a version close to what the CI uses: https://github.com/iree-org/iree/blob/c1e542d6370473244a8fa9178615cb8a6041b489/.pre-commit-config.yaml#L33-L41

Reply:
Thanks, I have set up pre-commit and it is very helpful.